model/database/registry.yaml

```diff
@@ -127,238 +127,181 @@ groups:
         examples: ["102", "ORA-17002", "08P01", "404"]
       - id: db.response.returned_rows
         type: int
-        stability: experimental
+        stability: development
         brief: Number of rows returned by the operation.
         examples: [10, 30, 1000]
-      - id: db.system
+      - id: db.system.name
         brief: The database management system (DBMS) product as identified by the client instrumentation.
         note: >
           The actual DBMS may differ from the one identified by the client.
-          For example, when using PostgreSQL client libraries to connect to a CockroachDB, the `db.system`
+          For example, when using PostgreSQL client libraries to connect to a CockroachDB, the `db.system.name`
           is set to `postgresql` based on the instrumentation's best knowledge.
         type:
           members:
             - id: other_sql
               value: "other_sql"
               brief: "Some other SQL database. Fallback only. See notes."
-              stability: experimental
-            - id: adabas
-              value: "adabas"
+              stability: development
+            - id: software_ag.adabas
+              value: "software_ag.adabas"  # trademark on 'adabas'
```

Comment:
maybe can remove the trademark comments for values that are now embedded under company name?